afex_integration/models/account.py

```diff
@@ -84,6 +84,38 @@ class AccountJournal(models.Model):
         string="AFEX Fees Expense Account",
         domain=[('deprecated', '=', False)],
         copy=False)
+    afex_direct_debit_journal_id = fields.Many2one(
+        'account.journal',
+        string="Direct Debit Journal",
+        copy=False)
+    afex_direct_debit = fields.Boolean(
+        string='Direct Debit by Default',
+        default=False,
+        copy=False)
+
+    aud_currency_id = fields.Many2one(
```

Comment:
Rather than writing hard code for aud_currency_id here, can we make it a boolean "can_direct_debit", and let the boolean value be dependent on the currency and check it - that way, if the available ways of DD expand, we just have to change the function....